packages/opencensus-web-instrumentation-zone/src/interaction-tracker.ts

```diff
@@ -186,6 +199,46 @@ export class InteractionTracker {
     });
   }
 
+  private interceptXHRTasks(task: AsyncTask) {
+    if (isTrackedTask(task) && task.target instanceof XMLHttpRequest) {
+      const rootSpan: RootSpan = task.zone.get('data').rootSpan;
+      const xhr = task.target as XHRWithUrl;
+      if (xhr.readyState === XMLHttpRequest.OPENED) {
+        const childSpan = rootSpan.startChildSpan({
+          name: parseUrl(xhr.__zone_symbol__xhrURL).pathname,
```

Comment:
Could you add a comment that `__zone_symbol__xhrURL` is set by the Zone monkey patch?